portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,33 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Portfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Inconsolata&family=Roboto:wght@300;400&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+        <div id = "intro">
+            <p>Hi I'm</p>
+            <div id="name">
+                <h2>Rebecca Sun</h2>
+            </div>
+            <p>CS major at Vanderbilt University</p>
+        </div>
+        <div id="about">
+            <p>About</p>
+        </div>
+      <div id="profiles">
+        <h3>Let's Connect:</h3>
+        <h4>LinkedIn</h4>
+        <img src="https://img.icons8.com/color/48/000000/linkedin.png"/>
+        <h4>Github</h4>
+        <img src="https://img.icons8.com/ios-filled/50/000000/github.png"/>
```

Comment:
Make sure to provide "alts" : https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback